### PR TITLE
Github API Post and Patch Testing

### PR DESCRIPTION
**Pull request by manuelq12**
New Test added: Post and Patch request tested in Github API.

### CODE REVIEW

File: test/GithubApi.Issue.test.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+const statusCode = require('http-status-codes');
+const chai = require('chai');
+
+const { expect } = chai;
+
+const urlBase = 'https://api.github.com';
+
+describe('Github Api Test', () => {
+  describe('Scenario: Creating Issues', () => {
+    describe('Given our Github account', () => {
```
Comment (saradrada) [inline test/GithubApi.Issue.test.js:11]: When describing don't use third person

File: test/GithubApi.Issue.test.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            .set('User-Agent', 'agent');
+        });
+
+        it('Then at least one public repository is expected', () => {
+          expect(userInfo.status).to.equal(statusCode.OK);
+          expect(userInfo.body.name).equal('Manuel Quintero');
+          expect(userInfo.body.public_repos).to.be.at.least(1);
+        });
+      });
+      describe('When a GET request is used to retrieve our repository information', () => {
```
Comment (saradrada) [inline test/GithubApi.Issue.test.js:26]: maybe add a line before this describe, so it's easier to read :)

File: test/GithubApi.Issue.test.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+              .set('User-Agent', 'agent');
+          });
+
+          it('Then the issue should have been created', () => {
+            expect(issueResponse.status).to.equal(statusCode.CREATED);
+            expect(issueResponse.body.id).to.not.equal('undefined');
+            expect(issueResponse.body.title).to.equal(query.title);
+            expect(issueResponse.body.body).to.equal(null);
+          });
+          describe('When a PATCH request is used to update the issue in our repository', () => {
```
Comment (saradrada) [inline test/GithubApi.Issue.test.js:59]: same here, maybe add a line before the describe

**Review by saradrada — CHANGES_REQUESTED** [review]

File: test/GithubApi.Issue.test.js (hunk 1 above)
Comment (manuelq12) [reply test/GithubApi.Issue.test.js:11]: Oki doki haha

File: test/GithubApi.Issue.test.js (hunk 2 above)
Comment (manuelq12) [reply test/GithubApi.Issue.test.js:26]: Yes ma'am

File: test/GithubApi.Issue.test.js (hunk 3 above)
Comment (manuelq12) [reply test/GithubApi.Issue.test.js:59]: Yes ma'am x2

**Review by saradrada — APPROVED** [review]

**Pull request merged**